src/components/Widget/components/FullScreenPreview/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, ReactNode } from 'react';
+import ReactDOM from 'react-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import usePreview from './usePreview';
+import usePortal from './usePortal';
+import './styles.scss';
+import { GlobalState } from '../../../../store/types';
+import { closeFullscreenPreview } from '../../../../store/actions';
+
+const close = require('../../../../../assets/close.svg') as string;
+const plus = require('../../../../../assets/plus.svg') as string;
+const minus = require('../../../../../assets/minus.svg') as string;
+const zoomIn = require('../../../../../assets/zoom-in.svg') as string;
+const zoomOut = require('../../../../../assets/zoom-out.svg') as string;
+
+export default function PdfFullScreen() {
```

Comment:
Should be the same name as the folder to maintain consistency, in this case `FullScreenPreview` 😄